scripts/tasks/size-audit.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const fs = require('fs');
+
+exports.generateSizeData = function() {
+  const projectName = path.basename(process.cwd());
+  const sizeFilePath = path.join(process.cwd(), 'dist', `${projectName}.json`);
+  if (fs.existsSync(path.dirname(sizeFilePath))) {
+    const result = {
+      chunks: analyzeChunks()
+    };
+    const fileContents = JSON.stringify(result);
+    fs.writeFileSync(sizeFilePath, fileContents);
+  }
+};
+
+function analyzeChunks() {

```

Comment:
It would be really nice to have some basic jsdoc comment here describing the file format you expect to see.